jupiterone/resource_question.go

```diff
@@ -144,6 +144,39 @@ func resourceQuestionRead(_ context.Context, d *schema.ResourceData, m interface
 	if err != nil {
 		return diag.Errorf("failed to read existing question: %s", err.Error())
 	}
+	err = d.Set("title", question.Title)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	err = d.Set("description", question.Description)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	err = d.Set("tags", question.Tags)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	convertedQueries, err := queriesToMapStringInterface(question.Queries)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	err = d.Set("query", convertedQueries)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	/*
+		TODO @zemberdotnet: read updates once compliance is actually supported in the
+		internal client
+		err = d.Set("compliance", question.Compliance)
+		if err != nil {
+			return diag.FromErr(err)
+		}
+	*/
```

Comment:
It doesn't look like the client properly supports the compliance object. This is what compliance looks like in the client:
```
compliance {
  type
  details {
    name
    description
  }
}
```
Go Type (reflects the terraform schema as well):
```go
type QuestionComplianceMetaData struct {
	Standard     string   `json:"standard"`
	Requirements []string `json:"requirements"`
	Controls     []string `json:"controls"`
}
```

Reply:
I'm hesitant to make the change to have this read updates and make the change in the client at the same time. I think it will be safer to maintain existing behavior for this and make these changes in a future PR.